lib/trento_web/plugs/otel.ex

```diff
@@ -0,0 +1,16 @@
+defmodule TrentoWeb.Plugs.Otel do
+  @moduledoc false
+  require OpenTelemetry.Tracer, as: Tracer
+
+  def init(default) do
+    default
+  end
+
+  def call(conn, _default) do
+    current_user = conn.assigns.current_user
+
+    Tracer.add_event("Otel", [{"user", current_user}])
```

Comment:
How this would appear in the traces (more customization/different manners of propagation are also additionally possible):

![image](https://github.com/trento-project/web/assets/18139046/e080e3f0-ada4-4f2d-af74-8dd08fc2fe04)